projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts

```diff
@@ -105,7 +113,7 @@ export class CaseFileViewFieldComponent implements OnInit, AfterViewInit, OnDest
           this.resetErrorMessages();
           this.reloadPage();
         }
-    });
+      });
```

Comment:
Spaces added here and line 79 for seemingly no reason (could be wrong)

Reply:
@connorpgpmcelroy This might be my IDE adding the spaces when I ran its "format document" feature. The linting is passing anyways.